akka-stream/src/main/scala/akka/stream/impl/Sinks.scala

```diff
@@ -73,36 +76,14 @@ private[akka] class PublisherSink[In](val attributes: Attributes, shape: SinkSha
    * subscription a VirtualProcessor would perform (and it also saves overhead).
    */
   override def create(context: MaterializationContext): (AnyRef, Publisher[In]) = {
-    val proc = new VirtualPublisher[In]
+    val proc = new VirtualPublisher[In]()
```

Comment:
I for one dont mind not having () if there's a new already.
Not that it really matters though, OK either way ;)